pubspec.yaml

```diff
@@ -28,6 +28,8 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
+  rxdart: ^0.27.1
```

Comment:
Why do we need rxdart ? I dont see it used in any of the files of this PR

Reply:
Because I thought that I would be implementing an <Observable> but then I didn't :D. So I just removed this dependency. Thank you.